package.json

```diff
@@ -19,7 +19,8 @@
         "rimraf": "^2.6.3",
         "ts-jest": "^24.0.1",
         "tslint-microsoft-contrib": "^6.1.0",
-        "typescript": "^3.4.1"
+        "typescript": "^3.4.1",
+        "license-check-and-add": "^2.3.6"
```

Comment:
Sort alphabetically like the rest? (in the future, if you use `npm install --save-dev package-name`, it will do this for you :) )

Reply:
Will update.